parser/crawler_current_events.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import re
+
+
+def crawl_page(edgar_13f_filing_detail_url_list, primary_doc_xml_list, infotable_xml_list):
```

Comment:
This function is doing way too much. Should be split into at least 3 functions. Each function should return a list rather than take in a list and modify it.